modules/auth/proc-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,15 @@
+[id="authenticating-with-the-guest-user"]
+= Authenticating with the Guest user
+
+To start exploring {product} features, do not worry about authentication: after the default installation, you can authenticate with the Guest user, and use all the application features.
+
+.Prerequisites
+* You installed {product-short} by using default values.
+See link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}#Install%20and%20Upgrade[Installing RHDH].
+
+.Procedure
+. Go to the {product-short} login page.
+. Log in with the Guest user account.
```

Comment:
Where do they find the Guest user account information? Or is there a button that says "Guest login" on the login page? It is unclear how they login with the "Guest user" account from this step.

Reply:
```suggestion
. To log in with the Guest user account, click **Enter** in the **Guest** tile.
```